src/POGOProtos/Networking/Requests/Messages/IncenseEncounterMessage.proto

```diff
@@ -2,6 +2,6 @@ syntax = "proto3";
 package POGOProtos.Networking.Requests.Messages;
 
 message IncenseEncounterMessage {
-	int64 encounter_id = 1;
+	fixed64 encounter_id = 1;
```

Comment:
Please add a note on the type, saying it's checked and never never change it again or something.